openapi/openapi.yaml

```diff
@@ -2084,7 +2074,7 @@ components:
                 has started.
             state:
               type: string
-              example: terminated
+              example: Running
```

Comment:
nitpick: can we keep these states lowercase? (they can be translated in thoth-common)

Reply:
Sure, I will add a tolower() in thoth-common :+1: